packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6911,8 +6908,6 @@ export default class Meeting extends StatelessWebexPlugin {
   async publishTracks(tracks: LocalTracks): Promise<void> {
     this.checkMediaConnection();
 
-    this.annotationInfo = tracks.annotationInfo;
```

Comment:
`this.annotationInfo` is also being read in `requestScreenShareFloor()` - now it will need to be read from `this.mediaProperties.shareVideoTrack` instead